scripts/src/osd/modules.lua

```diff
@@ -35,6 +39,30 @@ function addoptionsfromstring(str)
 	end
 end
 
+function addincluderootfromstring(str, EndPart)
+	if (str==nil) then return  end
+	for w in str:gmatch("%S+") do
+		if string.starts(w,"-I")==true then
```

Comment:
Comparing Booleans to true/false is an antipattern.

Reply:
Done